catalog/src/main/java/org/killbill/billing/catalog/CatalogUpdater.java

```diff
@@ -56,6 +56,17 @@
 public class CatalogUpdater {
 
     public static String DEFAULT_CATALOG_NAME = "DEFAULT";
+    
+    public static String INVALID_PLAN = "Plan doesn't exist or Plan is Invalid.";
+    
+    public static String INVALID_PRODUCT_NAME = "Product Name doesnt exist. Please provide valid Product Name.";
+    
+    public static String INVALID_PRICE = "Please check Amount and Currency. Amount should be greater than 0 and currency should be valid.";
+    
+    public static String BASE_PLAN_PRODUCTS_NOT_EMPTY = "Base Plan Products should not be empty.";
+    
+    public static String EXISTING_PRODUCTS_NOT_EMPTY = "Existing current product should not be empty.";
```

Comment:
This error message is really not clear - and misleading. The issue here is that the client input contains a list of available products associated with the add-on, and when the code check if such products exist it fails, so basically the list provided contains reference to invalid products.

Reply:
I have the error description to "Available base products contain invalid product.Please check."